engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -1864,7 +1865,7 @@ class IrToTruffle(
           scopeName,
           () => scopeInfo().scope,
           "case " + currentVarName,
-          frameInfo
+          if (frameInfoOption.isDefined) frameInfo else null
```

Comment:
There are various failures when running `ExecCompilerTest` indicating that certain `Function.Lambda` don't have `FramePointerMeta` (but neither `AliasAnalysis.Scope`). Here is the list of such functions:
```
Empty data for Some(IdentifiedLocation[location=Location[start=10961, end=10987], uuid=None]) in Array_Like_Helpers.enso code: i -> ((function) ((at) vector i))
Empty data for Some(IdentifiedLocation[location=Location[start=3667, end=4405], uuid=None]) in Array_Like_Helpers.enso code: i -> 

    result = ((function) i)
    ((if_then_else) ((not) ((is_error) result)) result 

        
            <internal-9> = (FORCE on_problems)
            case (FORCE <internal-9>) of
                Problem_Behavior.Ignore  -> 
                    

                        Nothing
                Problem_Behavior.Report_Error  -> 
                    

                        ((catch_primitive) result caught -> 


                            ((throw) Error ((Error) Map_Error i caught)))
                No_Wrap  -> (FORCE result)
                Problem_Behavior.Report_Warning  -> 
                    
                        with_error_maybe = ((if_then_else) ((>=) ((get) num_errors) ((MAX_MAP_WARNINGS) <internal-3056>)) Nothing 

                            ((catch_primitive) result caught -> 


                                ((attach) Warning caught Nothing)))
                        ((modify) num_errors <internal-6> -> ((+) <internal-6> 1))
                        (FORCE with_error_maybe))
Empty data for Some(IdentifiedLocation[location=Location[start=3959, end=4031], uuid=None]) in Array_Like_Helpers.enso code: caught -> 


    ((throw) Error ((Error) Map_Error i caught))
Empty data for Some(IdentifiedLocation[location=Location[start=4258, end=4325], uuid=None]) in Array_Like_Helpers.enso code: caught -> 


    ((attach) Warning caught Nothing)
Empty data for Some(IdentifiedLocation[location=Location[start=4364, end=4367], uuid=None]) in Array_Like_Helpers.enso code: <internal-6> -> ((+) <internal-6> 1)
Empty data for Some(IdentifiedLocation[location=Location[start=12378, end=12454], uuid=None]) in Vector.enso code: t -> ((Numeric_Input) Widget (minimum = 0) (maximum = ((-) ((length) t) 1)) (display = ((Always) Display)))
Empty data for Some(IdentifiedLocation[location=Location[start=14122, end=14213], uuid=None]) in Array_Like_Helpers.enso code: builder -> 


    ((map) vector elem -> 


        ((if_then) ((predicate) elem) ((append) builder elem)))
Empty data for Some(IdentifiedLocation[location=Location[start=14151, end=14213], uuid=None]) in Array_Like_Helpers.enso code: elem -> 


    ((if_then) ((predicate) elem) ((append) builder elem))
Empty data for Some(IdentifiedLocation[location=Location[start=7225, end=7265], uuid=None]) in Filter_Condition.enso code: (filter : Filter_Condition) -> (FORCE filter)
Empty data for Some(IdentifiedLocation[location=Location[start=7350, end=7359], uuid=None]) in Filter_Condition.enso code: <internal-1821> -> (FORCE filter)
Empty data for Some(IdentifiedLocation[location=Location[start=8043, end=8078], uuid=None]) in Vector.enso code: builders -> ((function) ((at) builders 0))
Empty data for Some(IdentifiedLocation[location=Location[start=10690, end=10741], uuid=None]) in Vector.enso code: <internal-3049> -> ((new) Builder initial_capacity propagate_warnings)
Empty data for Some(IdentifiedLocation[location=Location[start=7073, end=7113], uuid=None]) in Range.enso code: i -> ((function) ((+) ((start) self) ((*) i ((step) self))))
Empty data for Some(IdentifiedLocation[location=Location[start=10339, end=10528], uuid=None]) in Panic.enso code: caught_panic -> 

    error = ((payload) ((payload) caught_panic))
    ((if_then_else) ((not) ((is_a) error error_type)) ((throw) Panic ((payload) caught_panic)) 

        ((handler_resolved) error))
Empty data for Some(IdentifiedLocation[location=Location[start=11252, end=11292], uuid=None]) in Array_Like_Helpers.enso code: acc ix -> ((function) acc ((at) vector ix))
Empty data for Some(IdentifiedLocation[location=Location[start=12780, end=12782], uuid=None]) in Range.enso code: <internal-437> <internal-438> -> ((>=) <internal-437> <internal-438>)
Empty data for Some(IdentifiedLocation[location=Location[start=12790, end=12792], uuid=None]) in Range.enso code: <internal-439> <internal-440> -> ((<=) <internal-439> <internal-440>)
Empty data for Some(IdentifiedLocation[location=Location[start=12806, end=12997], uuid=None]) in Range.enso code: acc current -> 


    ((if_then_else) ((end_condition) current ((end) self)) acc 
        new_acc = ((function) acc current)
        ((go) new_acc ((+) current ((step) self))))
Empty data for Some(IdentifiedLocation[location=Location[start=8133, end=8144], uuid=None]) in Array_Like_Helpers.enso code: <internal-7> -> ((at) vector <internal-7>)
Empty data for Some(IdentifiedLocation[location=Location[start=9686, end=9712], uuid=None]) in Index_Sub_Range.enso code: i -> ((not) ((predicate) ((at) i)))
Empty data for Some(IdentifiedLocation[location=Location[start=2393, end=2409], uuid=None]) in Common.enso code: <internal-132> -> ((ends_with) <internal-132> "=_")
Empty data for Some(IdentifiedLocation[location=Location[start=2418, end=2429], uuid=None]) in Common.enso code: <internal-133> -> ((split) <internal-133> "=")
Empty data for Some(IdentifiedLocation[location=Location[start=2438, end=2444], uuid=None]) in Common.enso code: <internal-134> -> ((at) <internal-134> 0)
Empty data for Some(IdentifiedLocation[location=Location[start=21667, end=21775], uuid=None]) in Extensions.enso code: result -> 

    <internal-2807> = (FORCE result)
    case (FORCE <internal-2807>) of
        Nothing  -> []
        <internal-2878> -> [(FORCE result)]
Empty data for Some(IdentifiedLocation[location=Location[start=4723, end=4727], uuid=None]) in Error.enso code: x -> (FORCE x)
Empty data for Some(IdentifiedLocation[location=Location[start=4760, end=5078], uuid=None]) in Error.enso code: error_value -> 


    
        <internal-30> = ((is_a) error_value error_type)
        case (FORCE <internal-30>) of
            True  -> ((handler) error_value)
            False  -> 
                

                    
                        <internal-31> = ((look_for_wrapped_error) <internal-3084> error_type error_value)
                        case (FORCE <internal-31>) of
                            Maybe.None  -> self
                            Maybe.Some (inner_error) -> ((handler) inner_error)
```

Reply:
Possibly the check is no longer needed. Removed in 6b9f6f5 - let's see the result.